stock_picking_move_description/report/stock_picking_templates.xml

```diff
@@ -32,9 +32,23 @@
         >
             <th name="th_sm_description"><strong>Description</strong></th>
         </xpath>
+    </template>
 
+    <template
+        id="stock_report_delivery_has_serial_move_line_description"
+        inherit_id="stock.stock_report_delivery_has_serial_move_line"
+    >
+        <xpath
+            expr="//span[@t-field='move_line.product_id']/parent::td"
+            position="after"
+        >
+            <td>
+                <span t-field="move_line.move_id.description_picking"/>
+            </td>
+        </xpath>
```

Comment:
This addition should work, please test it:
```suggestion
        </xpath>
        <xpath expr="//span[@t-esc='description']" position="attributes">
            <attribute name="t-if">False</attribute>
        </xpath>        
```

This will prevent description_picking to be repeated, see:

https://github.com/odoo/odoo/blob/5ffafa7e71fc5de4974bcb326b2ea1eb5cb74d5a/addons/stock/report/report_deliveryslip.xml#L219-L224